third_party/github.com/cloudflare/circl/LICENSE

```diff
@@ -0,0 +1,57 @@
+Copyright (c) 2019 Cloudflare. All rights reserved.
```

Comment:
Running update-codegen on main to investigate these unexpected changes for the licenses.